runners/pvc_autoresizer.go

```diff
@@ -148,7 +148,8 @@ func (w *PVCAutoresizer) resize(ctx context.Context, pvc *corev1.PersistentVolum
 		return nil
 	}
 
-	increase, err := convertSizeInBytes(pvc.Annotations[ResizeIncreaseAnnotation], pvc.Spec.Resources.Limits.Storage().Value(), DefaultIncrease)
+	curReq := pvc.Spec.Resources.Requests[corev1.ResourceStorage]
+	increase, err := convertSizeInBytes(pvc.Annotations[ResizeIncreaseAnnotation], curReq.Value(), DefaultIncrease)
```

Comment:
It also seems reasonable for me to calculate the increase by multiplying the **limit** by the percent.
`limit * increase_ratio` is a easily predictable constant even when resizing occurs multiple times.

Anyway it should be described in the document to which the increase percent is applied.

Reply:
To increase size constantly, users can specify the concrete amount such as "30Gi".